llvm/docs/BitCodeFormat.rst

```diff
@@ -1006,93 +1006,29 @@ number of values representing the bytes of a null-terminated string. For
 attributes with a string argument (code 4), the *value* value is similarly a
 variable number of values representing the bytes of a null-terminated string.
 
-The integer codes are mapped to well-known attributes as follows.
+The integer codes are mapped to well-known attributes as described in the
```

Comment:
```suggestion
The integer codes are mapped to attributes as described in the
```
While we're looking at this, what does "well-known" mean in this context?

Reply:
Excellent question, I was just preserving the existing adjective but I agree it's unclear. I'll defer to someone with better knowledge of the area, I am happy to remove it if there is agreement.

Reply:
I guess that this is trying to distinguish pre-defined enum attributes from string attributes. Agree that just omitting the word would be clearer.

Reply:
Fixed in latest